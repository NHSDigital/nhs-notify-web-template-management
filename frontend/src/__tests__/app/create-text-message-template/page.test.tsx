/**
 * @jest-environment node
 */
import CreateSMSTemplatePage, {
  generateMetadata,
} from '@app/create-text-message-template/page';

jest.mock('@forms/SmsTemplateForm/SmsTemplateForm');

describe('CreateSMSTemplatePage', () => {
  beforeEach(jest.resetAllMocks);

  it('should render CreateSMSTemplatePage', async () => {
    generateMetadata();
    const page = await CreateSMSTemplatePage();

    expect(page).toMatchSnapshot();
  });
});
