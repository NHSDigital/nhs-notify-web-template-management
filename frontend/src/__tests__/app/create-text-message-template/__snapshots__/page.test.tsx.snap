// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateSMSTemplatePage should render CreateSMSTemplatePage 1`] = `
<SmsTemplateForm
  csrfToken="csrf-token"
  initialState={
    {
      "message": "",
      "name": "",
      "templateStatus": "NOT_YET_SUBMITTED",
      "templateType": "SMS",
    }
  }
/>
`;
